packages/components/src/date-time/time/index.tsx

```diff
@@ -61,6 +61,7 @@ export function TimePicker( {
 	currentTime,
 	onChange,
 	dateOrder: dateOrderProp,
+	withLabels = true,
```

Comment:
Should this prop be `hideLabelFromVision` like we have for all the controls.

Reply:
Nice, didn't know that was a common pattern in components. Pushed https://github.com/WordPress/gutenberg/pull/64267/commits/f099a4656342883336ad278050107326443cf015 to update to `hideLabelFromVision` that follows what other components do (e.g.: [border-control](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/border-control/border-control/component.tsx#L29)).